TEMPLATES.md

```diff
@@ -111,3 +111,10 @@ before gathering the artifacts.
 `ci-operator.openshift.io/always-show-output`:
 Will output the logs of all the containers in the pod, no matter what the exit code was.
 The value should be `true` to enable this feature.
+
+`ci-operator.openshift.io/always-show-container-output`:
```

Comment:
I'm wondering if `always-show-container-output` is the right name for the annotation (the name was reused from the previous feature, right?). It may be misleading because it looks similar to the `always-show-output` one but works differently. I don't like the `always` part - it's not true (always != when pod fails and only if container terminated) and it is what makes it similar to `always-show-output`.

I am not sure about a good name, though... `containers-logged-on-failure` ?